src/pages/home/Home.jsx

```diff
@@ -1,9 +1,104 @@
+                <h3 className="style-head"><span>Reading made</span>
+                    <span className="style_text-yellow"><span className="style-yellow">better</span></span></h3>
+                <p> <span className="style-description">
+                        Read more books
+                        <br/>
+                        Meet great people
+                        <br/>
+                        Engage in meaningful conversations
+                    </span></p>
+                <Link to="/product" className="btn-bookooze">Bookstore</Link>
+            </div>
+
+            <div className="books-slider">
+                <div>
+                    <img src={BackgroundIamge} className="style-image" alt=""/>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section className="icons-container">
+        <div className="icons">
+            <i className="fas fa-shipping-fast icons_for-bookhub"></i>
+            <div className="content">
+                <h3 className="port-shipping">Free shipping</h3>
+                <p className="facility_for-port">order over $100</p>
+            </div>
+        </div>
+
+        <div className="icons">
+            <i className="fas fa-lock icons_for-bookhub"></i>
+            <div className="content">
+                <h3 className="port-shipping">Secure payment</h3>
+                <p className="facility_for-port">100 secure payment</p>
+            </div>
+        </div>
+
+        <div className="icons">
+            <i className="fas fa-redo-alt icons_for-bookhub"></i>
+            <div className="content">
+                <h3 className="port-shipping">Easy returns</h3>
+                <p className="facility_for-port">10 days returns</p>
+            </div>
+        </div>
+
+        <div className="icons">
+            <i className="fas fa-headset icons_for-bookhub"></i>
+            <div className="content">
+                <h3 className="port-shipping">24/7 support</h3>
+                <p className="facility_for-port">call us anytime</p>
+            </div>
+        </div>
+    </section>
+    <section className="categories">
+    <div>{category.map(item => (
+      <div className='categoryImg' key={item.id}>
+          <img src={item.categoryImage} alt="category-img"/>
+          <h2>{item.categoryName}</h2>
+      </div>
+      
+))}</div>
+    </section>
+
+    <section className="deal">
+        <div className="deal-content">
+            <h3 className="deal-heading">Deal of the day</h3>
+            <h1 className="off-deal">upto 50% off</h1>
+            <p className="deal-desc">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Unde perspiciatis in atque
+                dolore tempora
+                quaerat at fuga dolorum natus velit.</p>
+            <a href="#" className="btn">Shop now</a>
+        </div>
+        <div className="image">
+            <img className="deal-image" src={require("../../assets/book13.jpg")} alt=""/>
```

Comment:
suggestion :
host the image on platforms like Cloudinary or Postimages and then access them by their resp. URL's
these images will take less time to load

Reply:
That's great thank you for the suggestion🤝